easybuild/easyconfigs/g/GCCcore/GCCcore-13.2.0.eb

```diff
@@ -62,7 +64,10 @@ builddependencies = [
 languages = ['c', 'c++', 'fortran']
 
 withisl = True
-withnvptx = True
+if ARCH == 'riscv64':
+    withnvptx = False
+else:
+    withnvptx = True
```

Comment:
@bedroge
```suggestion
# disable NXPTX on RISC-V
withnvptx = ARCH != 'riscv64'
```

But I would actually prefer doing this in the easyblock rather than hardcoding it in a bunch of easyconfig files...

Reply:
That makes sense. I'll wait for the build to complete to make sure that it works, and then I'll move this to the easyblock